src/components/common/Navbar/index.tsx

```diff
@@ -36,11 +44,7 @@ const NavScrollExample = () => {
               </NavLink>
             )}
             {me && (
-              <NavLink
-                className="nav-link"
-                to="/login"
-                onClick={() => me && logOut(me)}
-              >
+              <NavLink className="nav-link" to={to} onClick={handleClick}>
```

Comment:
Eliminá la prop to y cambia el componente por un Button de bootstrap. El resto queda igual. Vas a utilizar NavLink de react-router solo cuando necesites navegar entre diferentes rutas sin hacer refresh ni operaciones previas a esa navegación.
Cuando necesitas realizar algún tipo de tarea, utilizas botones normales, creas en handle click, submit, keyup o el que sea y una vez que terminas la tarea, si necesitas moverte, utilizas navigate